### PR TITLE
Update endpoint-dlp-using.md

### PR DESCRIPTION
**Pull request by msbemba**
Added subscription requirement for endpoint DLP. @chrfox  This is assuming that the licenses needed have not changed with Microsoft purview.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9082

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/endpoint-dlp-using.md [hunk 1]
```diff
@@ -32,6 +32,22 @@ To help familiarize you with Endpoint DLP features and how they surface in DLP p
...
+Before you get started with Endpoint DLP, you should confirm your [Microsoft 365 subscription](https://www.microsoft.com/microsoft-365/compare-microsoft-365-enterprise-plans?rtc=1) and any add-ons. To access and use Endpoint DLP functionality, you must have one of these subscriptions or add-ons.
+
+- Microsoft 365 E5
+- Microsoft 365 A5 (EDU)
+- Microsoft 365 E5 compliance
+- Microsoft 365 A5 compliance
+- Microsoft 365 E5 information protection and governance
+- Microsoft 365 A5 information protection and governance
+
+for full licensing details, see [Microsoft 365 licensing guidance for information protection](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#information-protection-data-loss-prevention-for-exchange-online-sharepoint-online-and-onedrive-for-business)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-using.md]:
```suggestion
For full licensing details, see [Microsoft 365 licensing guidance for information protection](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#information-protection-data-loss-prevention-for-exchange-online-sharepoint-online-and-onedrive-for-business).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thnaks.

**Pull request merged**